<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.000">
  <properties />
            <testcase name="loginTestCase" time="0.000" classname="com.w2aTestCases.LoginTest">
                <failure type="org.testng.TestNGException"
                  message="
Data Provider public java.lang.Object[][] com.w2aTestCases.LoginTest.getData() returned a null value"
              >
<![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[][] com.w2aTestCases.LoginTest.getData() returned a null value
  org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:133)
  org.testng.internal.Parameters.handleParameters(Parameters.java:706)
  org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
  org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:924)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:648)
  org.testng.TestRunner.run(TestRunner.java:505)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
  org.testng.SuiteRunner.run(SuiteRunner.java:364)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
  org.testng.TestNG.runSuites(TestNG.java:1049)
  org.testng.TestNG.run(TestNG.java:1017)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
